addon/components/bs3/bs-form.js

```diff
@@ -1,9 +1,12 @@
-import { computed } from '@ember/object';
+import { computed, assert } from '@ember/object';
 import Form from 'ember-bootstrap/components/base/bs-form';
 
 export default Form.extend({
   layoutClass: computed('formLayout', function() {
     let layout = this.get('formLayout');
+    let supportedTypes = ['vertical', 'horizontal', 'inline'];
+    assert('must provide a valid `formLayout` attribute.',
+      supportedTypes.includes(this.get('formLayout')));
```

Comment:
I think older browsers don't have `.includes`. You should either wrap the array in a `Ember.A()` or use plain `indexOf(...) !== -1`. Also you can reuse `layout` instead of getting this again.